ape_safe/accounts.py

```diff
@@ -677,7 +674,7 @@ def skip_signer(signer: AccountAPI):
         # NOTE: It is okay to have less signatures, but it never should fetch more than needed
         signers = [x for x in available_signers if x.address not in sigs_by_signer]
         if signers:
-            new_signatures = get_signatures(safe_tx_hash, signers)
```

Comment:
Same here, is the value of `safe_tx_hash` still in use?

Reply:
used on L724